### PR TITLE
Added the LDAP component

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: translations/component.en.xliff [hunk 1]
```diff
@@ -201,6 +201,15 @@
                 <target />
             </trans-unit>
 
+            <trans-unit id="ldap.summary">
+                <source>ldap.summary</source>
+                <target>Provides a LDAP client for PHP on top of PHP's ldap extension.</target>
```
Comment (xabbuh) [inline translations/component.en.xliff]: [...] an LDAP client [...]

**Comment by xabbuh** [comment]
@OskarStark By the way, there's a lot of other stuff not completely translated into German.

File: translations/component.en.xliff (hunk 1 above)
Comment (javiereguiluz) [inline translations/component.en.xliff]: Fixed. Thanks.

**Comment by OskarStark** [comment]
Will have a look

btw. what is recommended?
```xml
            <trans-unit id="browserkit.description">
                <source>browserkit.description</source>
                <target></target>
            </trans-unit>
```

```xml
            <trans-unit id="config.description">
                <source>config.description</source>
                <target />
            </trans-unit>
```

or

```xml
            <trans-unit id="config.description">
                <source>config.description</source>
                <target> config.description</target>
            </trans-unit>
```

the files are mixed...

**Comment by javiereguiluz** [comment]
@OskarStark merged!

**Pull request closed without merging**